### PR TITLE
ReturnPrime trace added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Issue #6185

### CODE REVIEW

File: src/technologies/r.json [hunk 1]
```diff
@@ -1426,6 +1426,19 @@
     "scriptSrc": "\\.returngo\\.ai/",
     "website": "https://returngo.ai"
   },
+  "ReturnPrime": {
+    "cats": [
+      102
+    ],
+    "description": "Most loved and only 5 star rated RMA app to manage returns, exchanges and refunds for your shopify store.",
```
Comment (nurbek91) [inline src/technologies/r.json:1433]: "description": "Return Prime is an application to manage returns for Shopify stores."

File: src/technologies/r.json [hunk 2]
```diff
@@ -1426,6 +1426,19 @@
     "scriptSrc": "\\.returngo\\.ai/",
     "website": "https://returngo.ai"
   },
+  "ReturnPrime": {
```
Comment (nurbek91) [inline src/technologies/r.json:1429]: Return Prime

File: src/technologies/r.json [hunk 3]
```diff
@@ -1426,6 +1426,19 @@
...
+    "cats": [
+      102
+    ],
```
Comment (nurbek91) [inline src/technologies/r.json:1432]:
```
"cats": [
      100,
      102
    ],
```

File: src/technologies/r.json [hunk 4]
```diff
@@ -1426,6 +1426,19 @@
...
+    "description": "Most loved and only 5 star rated RMA app to manage returns, exchanges and refunds for your shopify store.",
+    "icon": "return-prime.png",
+    "pricing": [
+      "freemium",
+      "recurring"
+    ],
```
Comment (nurbek91) [inline src/technologies/r.json:1438]:
```
"pricing": [
      "freemium",
      "low",
      "recurring"
    ],
```

File: src/technologies/r.json [hunk 5]
```diff
@@ -1426,6 +1426,19 @@
...
+    "scriptSrc": "//return-prime-proxy-prod\\.s3[^ ]*\\.amazonaws.com/",
```
Comment (nurbek91) [inline src/technologies/r.json:1439]: `"scriptSrc": "//return-prime-proxy-prod\\.s3[^ ]*\\.amazonaws\\.com/",`

Comment (nurbek91) [reply src/technologies/r.json:1439]: `"requires": "Shopify",`

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR

File: src/technologies/r.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/r.json:1433]: done

**Comment by dhavalveera** [comment]
@nurbek91 done all the improvements you stated above

**Review by nurbek91 — APPROVED** [review]
thanks!

**Pull request merged**